db/schema.rb

```diff
@@ -35,8 +35,8 @@
     t.string   "type"
     t.date     "needed_date"
     t.string   "status"
-    t.datetime "created_at",         null: false
-    t.datetime "updated_at",         null: false
+    t.datetime "created_at",                     null: false
+    t.datetime "updated_at",                     null: false
```

Comment:
@another wacko alignment thing - doesn't really matter here, and probably came from automagic rails stuff anyway.